parser.py

```diff
@@ -0,0 +1,34 @@
+import bs4
+import requests
+
+# вставить адрес с сайта лостфильм.тв, раздел гид по сериям
+httpAddress = 'https://www.lostfilm.tv/series/Rome/seasons'
+
+# название сериала из адреса
+seriesName = ((str(httpAddress)).split('/')[-2])
```

Comment:
Здесь переменная `httpAddress` изначально задается как строка -- зачем еще раз приводить ее к строке?
В модуле `urllib.parse` есть функции для того, чтобы парсить url'ы. Здесь хорошо, что url изначально задан и правильный, но если на вход будет подано что-то типа `https://www.lostfilm.tv/series/Rome/seasons#hello/world` (тоже валидный url), то все сломается
```suggestion
from urllib.parse import urlparse

url_data = urlparse(httpAddress)
series_name = url_data.path.split('/')[-2]
```